storybook/RechartsTheme.ts

```diff
@@ -0,0 +1,36 @@
+import { create } from '@storybook/theming/create';
+
+export default create({
```

Comment:
Wonderful, this is great. Thank you! 
I am not reviewing each and every value here, simply because this PR is a big step in the right direction and sets up the configuration. 

We can independently iterate on the exact values to be used.